temba/channels/types/telegram/type.py

```diff
@@ -46,11 +47,37 @@ def deactivate(self, channel):
         bot = telegram.Bot(config['auth_token'])
         bot.delete_webhook()
 
+    def get_quick_replies(self, metadata, post_body):
+        metadata = json.loads(metadata)
+        quick_replies = metadata.get('quick_replies', None)
+        url_buttons = metadata.get('url_buttons', None)
+        replies = []
+        keyboard_type = None
+
+        if quick_replies:
+            for reply in quick_replies:
+                replies.append([dict(text=reply.get('title'), callback_data=reply.get('payload'))])
+            keyboard_type = 'keyboard'
+        elif url_buttons:
```

Comment:
Guys, please, we made it clear what we were willing to accept in this PR and we explicitly said  quick replies only and with only the payload. Don't act as if this is new. Let's do this in steps as we first asked.